manim/animation/composition.py

```diff
@@ -104,8 +103,9 @@ def _setup_scene(self, scene) -> None:
             anim._setup_scene(scene)
 
     def finish(self) -> None:
-        for anim in self.animations:
-            anim.finish()
+        self.interpolate(1)
+        self.anims_begun[:] = False
+        self.anims_finished[:] = False
```

Comment:
It shouldn't really matter I guess, but is this supposed to be `True`?

Reply:
Right, I changed it now.